app/assets/stylesheets/application.scss

```diff
@@ -111,12 +111,13 @@ a {
 	background-color: #14171a;
 	color: white;
 	position: relative;
-	padding-bottom: 0.5rem;
+	padding-bottom: 0.25rem;
+	// padding-top: 0.5rem;
 	left: 0;
 	bottom: 0;
-	right:0;
+	right: 0;
 	width: 100%;
-	// padding-top: 1rem;
+	padding-top: 0.5rem;
```

Comment:
very cool using rem`s. It'd be awesome to apply this to font-sizes for accessibility!